src/python/pants/option/alias.py

```diff
@@ -47,10 +47,12 @@ class CliOptions(Subsystem):
                 [cli.alias]
                 green = "fmt lint check"
                 all-changed = "--changed-since=HEAD --changed-dependents=transitive"
+                --troubleshoot = "--test-debug --test-force --test-timeout-maximum=300"
 
 
             This would allow you to run `{bin_name()} green all-changed`, which is shorthand for
```

Comment:
```suggestion
            This would allow you to run `{bin_name()} green --all-changed`, which is shorthand for
```

Reply:
That's great, let's reuse an existing one!